### PR TITLE
Add German localization

### PR DESCRIPTION
**Pull request by WAUthethird**
This is a German-language localization file, created by kruemel in the Discord.

I am unaware as to the legitimacy and/or accuracy of the translation, so some review may be needed, though I am not bilingual. If you can, please point out any errors.

### CODE REVIEW

**Comment by LocherGit** [comment]
Hello, my name is Locher and I reviewed the german localization attempt of my peers.

I used gbstudio extensively for the last weeks and therefore have enough contextual information to be confident in my proposed changes. I have used most if not all script functions that gbstudio offers and I am familiar with the different side windows within gbstudio.

My proposes are double-checked in the situations and contexts they were used for and I respected the conformity of how the Script functions are titled in the english version so that german users have an easier time searching for them via searchbar. Many terms were ambiguous and my peers seem to have misunderstood their use contexts so I took the liberty to correct them.

I am sure that german users of gbstudio will have an enjoyable time with gbstudio with these changes implemented.

**Comment by WAUthethird** [comment]
Thanks for the review @LocherGit! It's been implemented into the file.

File: src/lang/de.json [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+  "TOOL_ADD_SCENE_LABEL": "Szene hinzufügen",
+  "TOOL_ERASER_LABEL": "Radierer",
+  "TOOL_COLLISIONS_LABEL": "Kollisionen",
+
+  "// 5": "Editors --------------------------------------------------",
+  "SIDEBAR_NAVIGATION": "Navigation",
+  "SIDEBAR_STARTING_SCENE": "Startszene",
+  "SIDEBAR_SCENE_START_SCRIPT": "Szenenstart-Skript",
+  "SIDEBAR_ACTOR_SCRIPT": "Darsteller-Skript",
+  "SIDEBAR_TRIGGER_SCRIPT": "Auslöser-Script",
```
Comment (LocherGit) [inline src/lang/de.json]:
```suggestion
  "SIDEBAR_TRIGGER_SCRIPT": "Auslöser-Skript",
```

File: src/lang/de.json [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+  "SIDEBAR_ADD_EVENT": "Ereignis hinzufügen",
+
+  "// 6": "Form Fields ----------------------------------------------",
+  "FIELD_NAME": "Name",
+  "FIELD_AUTHOR": "Autor",
+  "FIELD_NOTES": "Notizen",
+  "FIELD_ADD_NOTES": "Notizen hinzufügen",
+  "FIELD_TYPE": "Typ",
+  "FIELD_ACTOR_INVOKE": "Führt den folgenden Darsteller-Skript aus",
```
Comment (JanSchneider1) [inline src/lang/de.json]: Shouldn't that be called "Führt **das folgende ** Darsteller-Skripts aus"?

Comment (JanSchneider1) [inline src/lang/de.json]:
```suggestion
  "FIELD_ACTOR_INVOKE": "Führt das folgende Darsteller-Skript aus",
```

**Pull request merged**